cmd/create/cluster/cmd.go

```diff
@@ -359,6 +359,21 @@ func run(cmd *cobra.Command, _ []string) {
 	}()
 	ocmClient := ocmConnection.ClustersMgmt().V1()
 
+	tempAWSClient := aws.GetAWSClientForUserRegion(reporter, logger)
+
+	creator, err := tempAWSClient.GetCreator()
+	if err != nil {
+		reporter.Errorf("Unable to get IAM credentials: %v", err)
+		os.Exit(1)
+	}
+
+	// Creating non STS clusters is not supported
+	if creator.IsSTS && args.roleARN == "" {
+		reporter.Errorf("IAM STS credentials can only create STS clusters --role-arn cannot be blank")
+		reporter.Errorf("You are AWS credentials returning an STS ARN: %s", creator.ARN)
```

Comment:
`You are AWS credentials` -> `Your AWS credentials are`

Reply:
If I told you my linting failures were all spelling would you believe me?

Reply:
Fixed.

Reply:
I have a code review checklist that includes:
- [ ] review spelling mistakes on @jharrington22's code